### PR TITLE
Fix errors in debug bulding

### PR DESCRIPTION
**Pull request by PeterWangIntel**
It's harmeless for release build.

### CODE REVIEW

**Comment by PeterWangIntel** [comment]
@d0ngjun

File: Source/web/InspectorEmulationAgent.cpp [hunk 1]
```diff
@@ -42,9 +42,11 @@ InspectorEmulationAgent::~InspectorEmulationAgent()
 void InspectorEmulationAgent::restore()
 {
     ErrorString error;
```
Comment (d0ngjun) [inline Source/web/InspectorEmulationAgent.cpp]: This unused variable will cause warning or error.

File: Source/web/InspectorRenderingAgent.cpp [hunk 2]
```diff
@@ -36,11 +36,13 @@ InspectorRenderingAgent::InspectorRenderingAgent(WebViewImpl* webViewImpl)
 void InspectorRenderingAgent::restore()
 {
     ErrorString error;
```
Comment (d0ngjun) [inline Source/web/InspectorRenderingAgent.cpp]: Same as above.

File: Source/web/InspectorEmulationAgent.cpp (hunk 1 above)
Comment (PeterWangIntel) [inline Source/web/InspectorEmulationAgent.cpp]: yes.thx

File: Source/web/InspectorRenderingAgent.cpp (hunk 2 above)
Comment (PeterWangIntel) [inline Source/web/InspectorRenderingAgent.cpp]: yes, thx

**Comment by d0ngjun** [comment]
LGTM

**Pull request merged**